packages/compass-web/src/entrypoint.tsx

```diff
@@ -268,11 +269,11 @@ const CompassWeb = ({
       enableAggregationBuilderRunPipeline: true,
       enableAggregationBuilderExtraOptions: true,
       enableImportExport: false,
-      enableGenAIFeatures: false,
+      enableGenAIFeatures: true,
```

Comment:
Let's default the org and project settings to false (they'll be overridden by what we pass in from initial preferences in mms later).
```suggestion
      enableGenAIFeatures: true,
      enableGenAIFeaturesAtlasProject: false,
      enableGenAISampleDocumentPassingOnAtlasProject: false,
      enableGenAIFeaturesAtlasOrg: false,
```